package.json

```diff
@@ -44,5 +44,24 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/nationalbankbelgium/stark.git"
+  },
+  "devDependencies": {
+    "husky": "0.14.3",
+    "lint-staged": "7.0.0",
+    "prettier": "1.10.2",
+    "stylelint": "8.2.0",
+    "stylelint-config-prettier": "2.1.0",
+    "tslint": "5.8.0",
```

Comment:
I think we should take the chance to upgrade TSLint to the latest version and see if it is still working fine with prettier